packages/editor-ui/src/components/NDVDraggablePanels.vue

```diff
@@ -98,6 +98,7 @@ export default mixins(debounceHelper).extend({
 	},
 	mounted() {
 		this.setTotalWidth();
+		this.ndvStore.setCurrentNodePaneType(this.currentNodePaneType);
```

Comment:
is the right place for this? ideally this would be a derived property set in the store..  though might not be simple to refactor this.. if not simple, then make sure you add a watch..
for example, what happens if you jump from one node to another without going back to canvas, then this would not be the right value
here's an example workflow, run this, open set node, click on see error, that switches to previous node
```
{
  "meta": {
    "instanceId": "27cc9b56542ad45b38725555722c50a1c3fee1670bbb67980558314ee08517c4"
  },
  "nodes": [
    {
      "parameters": {
        "options": {}
      },
      "id": "2257a530-3872-468c-8f98-e32a81abcdb8",
      "name": "Set",
      "type": "n8n-nodes-base.set",
      "typeVersion": 1,
      "position": [
        680,
        300
      ]
    },
    {
      "parameters": {
        "rule": {
          "interval": [
            {
              "field": "months",
              "monthsInterval": -1
            }
          ]
        }
      },
      "id": "365b3f4f-e114-4d09-ac7c-f24772da7e21",
      "name": "Schedule Trigger",
      "type": "n8n-nodes-base.scheduleTrigger",
      "typeVersion": 1,
      "position": [
        460,
        300
      ]
    }
  ],
  "connections": {
    "Schedule Trigger": {
      "main": [
        [
          {
            "node": "Set",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  }
}
```